int32 vehicle_id
float64 range
float64 angle
bool radar_vis